sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveQlSuite.scala

```diff
@@ -208,4 +208,22 @@ class HiveQlSuite extends SparkFunSuite with BeforeAndAfterAll {
         |USING 'cat' AS (`thing1` int, `thing2` string) FROM `default`.`parquet_t1`) AS t
       """.stripMargin)
   }
+
+  test("use backticks in output of Generator") {
+    val plan = parser.parsePlan(
+      """SELECT `gentab2`.`gencol2`
+        |FROM `default`.`src`
+        |LATERAL VIEW explode(array(array(1, 2, 3))) `gentab1` AS `gencol1`
+        |LATERAL VIEW explode(`gentab1`.`gencol1`) `gentab2` AS `gencol2`
+      """.stripMargin)
```

Comment:
nit: please follow the code style of other tests, e.g. https://github.com/dilipbiswal/spark/blob/SPARK-13698/sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveQlSuite.scala#L195-L199